messages/lightning.preview.site.md

```diff
@@ -1,38 +1,35 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+Enable Lightning Preview to see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when Lightning Preview detects source code changes.
+
+When you edit local files with Lightning Preview enabled, your LWCs reflect the following changes in your org. To apply any other local changes not listed here, you have to deploy them to your org using the `project deploy start` command.
 
 - Basic HTML and CSS edits
-- Importing new CSS
-- Javascript edits in-service component library
-- Javascript method changes in the LWC component that don't alter its public API.
+- Importing new CSS-only LWCs
+- JS edits in-service component library
+- JS method changes that don't affect the LWC's public API
 
-Other local changes may require deployment to your org. However, changes made directly in your org (like modifying component properties) are immediately live and won't show in your local files until you retrieve them from the org.
+When you make changes directly in your org, (like saving new component properties,) they're automatically deployed to your live site. To update your local version of the site with those changes, you have to retrieve them from your org using the `project retrieve start` command.
 
 # flags.name.summary
 
-Specify the site name for preview.
-
-# flags.name.description
-
-The site name needs to match the name of a site on the current org. Example: "B2C CodeCept"
+Name of the Experience Cloud site to preview. It has to match a site name from the current org.
 
-# examples
+# flags.target-org.summary
 
-- <%= config.bin %> <%= command.id %> --name MySite
+Username or alias of the target org. Not required if the `target-org` configuration variable is already set.
 
 # flags.debug.summary
 
-Debug SSR.
+Enable Node Inspector to debug server-side rendering.
 
-# flags.debug.description
-
-Debug Description.
-
-# flags.target-org.summary
+# examples
 
-undefined
+- Preview the site "Partner Central" from the org "myOrg":
+  <%= config.bin %> <%= command.id %> --name "Partner Central" --target-org myOrg
+- Preview the site "Partner Central" from the org "myOrg" with Node Inspector enabled:
+  <%= config.bin %> <%= command.id %> --name "Partner Central" --target-org myOrg --debug
```

Comment:
@lpomerleau do we have existing public SSR debugging documentation we can point at?

Reply:
Debugging is still TBD on our end - will post exact instructions once I have them